src/test/java/com/ericsson/ei/jmespath/test/TestIncompleteJmesPathFilter.java

```diff
@@ -0,0 +1,71 @@
+/*
+   Copyright 2018 Ericsson AB.
+   For a full list of individual contributors, please see the commit history.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+ */
+package com.ericsson.ei.jmespath.test;
+
```

Comment:
This class tests is more function tests. A unit test tests only one specific function where unit test provides a specific input and expects an expected output from the specific function.
If other thinks its ok to test on this level without a specific unit test for each functions, then I guess its ok.

Reply:
I talked with Vali and he said that existing tests are enough.